mulighetsrommet-api/src/test/kotlin/no/nav/mulighetsrommet/api/fixtures/MulighetsrommetTestDomain.kt

```diff
@@ -12,7 +12,7 @@ import no.nav.mulighetsrommet.database.FlywayDatabaseAdapter
 import no.nav.mulighetsrommet.domain.dbo.TiltakstypeDbo
 
 data class MulighetsrommetTestDomain(
-    val enheter: List<NavEnhetDbo> = listOf(NavEnhetFixtures.IT),
+    val enheter: List<NavEnhetDbo> = listOf(NavEnhetFixtures.IT, NavEnhetFixtures.Innlandet, NavEnhetFixtures.Oslo),
```

Comment:
Hvilke tester trenger vi denne endringen for? Synes det blir mer ryddig å sette dette eksplisitt for den testcasen som _faktisk_ trenger Innlandet og Oslo.

Reply:
Har forøvrig på agendaen og gå over å rydde litt ifm. setup-koden her på tvers av tester. Det kan jo hende at en default "TestDomain"-instanse er bedre med Innlandet og Oslo enn IT, men tror uansett det er en fordel å holde defaulten til et minimum og heller gjøre en setup som step 1 i de testene som faktisk har behov for dataen. Blir mer eksplisitt, og ytelse blir kanskje bittelitt bedre også.

Reply:
Godt poeng, jeg kan få satt det opp i selve testen istedenfor